# Introdução à Ciência de Dados - UFPB
Professor: Yuri Malheiros

## Medidas de Centralidade e Dispersão

A partir do dataset `movies.csv` calcule a média, mediana e desvio padrão para as colunas:

In [2]:
import pandas as pd
movies = pd.read_csv("movies.csv")

### US Gross

In [3]:
movies["US Gross"].mean()

44002085.163744524

In [4]:
movies["US Gross"].median()

22019465.5

In [5]:
movies["US Gross"].var()

3913166983182700.5

### Production Budget

In [6]:
movies["Production Budget"].mean()

31069171.4484375

In [7]:
movies["Production Budget"].median()

20000000.0

In [8]:
movies["Production Budget"].var()

1266357235689725.0

### Running Time

In [10]:
movies["Running Time (min)"].mean()

110.19354838709677

In [11]:
movies["Running Time (min)"].median()

107.0

In [12]:
movies["Running Time (min)"].var()

406.8697927793207

### IMDB Rating

In [13]:
movies["IMDB Rating"].mean()

6.283467202141901

In [16]:
movies["IMDB Rating"].median()

6.4

In [15]:
movies["IMDB Rating"].var()

1.5682300903199902

### IMDB Votes

In [17]:
movies["IMDB Votes"].mean()

29908.64457831325

In [18]:
movies["IMDB Votes"].median()

15106.0

In [19]:
movies["IMDB Votes"].var()

2019386306.1193645

### Análise dos Resultados

Quais colunas tiveram valores muito diferentes entre média e mediana? Quais os seus desvios padrão?

Us gross com o desvio padrão: 3913166983182700.5;
Prodution Budgets com o desvio padrão: 1266357235689725.0;
IMDB Votes com desvio padrão de: 2019386306.1193645

Que conclusões podemos tirar desses resultados?

concluo que existem filmes com um us gross muito "baixo" e outros "altos", que o valor investido em algumas produções são muito maiores que de outros por isso um grande desvio padrão, e os votos entre o melhor e o pior filme tambem com alta variancia.